internal/pkg/fakeroot/fakeroot.go

```diff
@@ -25,11 +25,11 @@ const (
 	// SubGIDFile is the default path to the subgid file.
 	SubGIDFile = "/etc/subgid"
 	// validRangeCount is the valid fakeroot range count.
-	validRangeCount = 65536
+	validRangeCount = uint32(65536)
 	// StartMax is the maximum possible range start.
-	startMax = 4294967296 - validRangeCount
+	startMax = uint32(4294901760)
```

Comment:
I would suggest you keep the arithmetic from before just to make it a little clear where the number is coming from.

I would even suggest writing this as:

```
validRangeCount = uint32(1 << 16)
startMax = uint32(1 << 32) - validRangeCount
```

Reply:
sorry, that doesn't work, as 1 << 32 will overflow uint32.

There's the very awkward `uint32(1 << 32 - 1) - validRangeCount + 1` or the other option of `^uint32(0) - validRangeCount + 1`.

Reply:
Are you training for a code obfuscation contest ?